src/page_store/jobs/gc.rs

```diff
@@ -71,23 +71,70 @@ where
     }
 
     async fn gc(&mut self, version: &Arc<Version>) {
+        // Reclaimate deleted files in `cleaned_files`.
```

Comment:
```suggestion
        // Reclaim deleted files in `cleaned_files`.
```

I think "reclaimate" is not a word :)